### DISCUSSION TITLE
Layout Variables from Component

### DISCUSSION THREAD

**Discussion by lukas-staab** [livewire/livewire#8479] [7. Bugs (archived)] [answered]
### Livewire version

v3.5.0

### Laravel version

v10.48

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Using Layout Component Class in Livewire conf ` 'layout' => \App\View\Components\Layout::class,`
2. Have a variable given to View in Layout Class like e.g. App Version: 
``` php 
public function render()
    {
        return view('components.layouts.index', [
            'version' => config('app.version')
        ]);
    }
```

Expected Result: Livewire uses the view variable from the Layout renderer method

Given Result with livewire: `$version not defined`. But it works in normal blade templates, which use `<x-layout>...</x-layout>` syntax

However: defining a class variable in the component, and setting it in the constructor, does give the expected result (it works) 

Possible Solutions: using the variables, given in layout render method in livewire or instead documenting that view variables are not respected by livewire, even if including the Component Class and not only the template path

### Repository or Wirebox URL

https://github.com/lukas-staab/livewire-layout-minimal

### Additional information

I found some quite old discussions which go in the same direction but are not the same, therefore i started a new one

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PerryvanderMeer** [comment, marked as answer]
Fix: https://github.com/livewire/livewire/pull/8480

**Reply by lukas-staab** [reply]
Thanks for the quick fix!
